code/tyler/django/tyler_django_lab-2/blog_proj/templates/users/login.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %} {% block content %}
+<h1>Login</h1>
+<form action="{% url 'users_app:login' %}" method="POST">
+  <label for="username">Username</label>
+  <input type="text" name="username" id="username" />
+  <label for="password">Password</label>
+  <input type="text" name="password" id="password" />
+  {%csrf_token%}
+  <input type="submit" value="submit" />
+</form>
+
```

Comment:
Could use a link to register on this page in case a user gets redirected here from another view.

Reply:
I would love to see more styling on the register/login forms. At least center the form on the page, give it some padding and a link from one form to another. Need an account? Register! Have an account? Log in!

Please apply the Bootstrap classes for styling forms and their submit buttons.